src/api/api.js

```diff
@@ -0,0 +1,52 @@
+import { cardData } from "../assets/data/cardData"
+
+export const deckUrl = 'https://deckofcardsapi.com/api/deck/new/'
+export const drawUrl = (id, remaining) => `https://deckofcardsapi.com/api/deck/${id}/draw/?count=${remaining}`
+
+export const fetchDeck = async () => {
+	const { deck_id, remaining } = await fetchApi(deckUrl)
+	const cards = fetchApi(drawUrl(deck_id, remaining)).then(({ cards }) => {
+		return cleanCards(cards)
+	})
+	return cards
+}
+
+export const fetchApi = async (url) => {
+	return await fetch(url).then((response) => response.json()).catch((error) => console.error('Can not reach server.'))
+}
+
+export const cleanCards = (cards) => {
```

Comment:
and here

Reply:
This and the other ones were because I did use them as exports until I decided to merge them 😅 forgot to clean up!!